antispam_bee.php

```diff
@@ -255,6 +255,20 @@ public static function init() {
 							'set_orderby_query',
 						)
 					);
+					add_action(
+						'restrict_manage_comments',
+						array(
+							'Antispam_Bee_Columns',
+							'filter_columns',
```

Comment:
Warum haben die Filter & Action keine Prio?

Reply:
If you don't set a priority, it defaults to `10`, so I do not see a problem there.

Reply:
its a bit more consistent but i'm fine.